docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: "3.9"
+
+networks:
+  nubedb:
+    external: false
+
+services:
+  node1:
```

Comment:
node1, container name node1, environment node1. Could there be a way to do it automatically?

Reply:
Will fix in a new issue: https://github.com/narvikd/nubedb/issues/12